### PR TITLE
[ci skip] Replacing ActiveRecord::Base with ApplicationRecord everywhere in guides

### PR DESCRIPTION
**Pull request by akshay-vishnoi**

### CODE REVIEW

**Comment by dhh** [comment]
Wups, just committed the other similar PR. Can you rebase and see if we missed anything in that?

**Comment by akshay-vishnoi** [comment]
@dhh - I added PR first. :smile: Its okay. I have rebased the PR. Please check.

**Comment by dhh** [comment]
These replace too much. Maybe of these uses are not ApplicationRecord specific. Actually, looking over them, none of these should be changed. They're all about framework features. Thanks for clarifying that to make it clear, though.

File: guides/source/plugins.md [hunk 1]
```diff
@@ -265,7 +265,7 @@ end
 
 # test/dummy/app/models/application_record.rb
 
-class ApplicationRecord < ActiveRecord::Base
+class ApplicationRecord < ApplicationRecord
```
Comment (akshay-vishnoi) [inline guides/source/plugins.md]: @dhh - Shouldn't we include these changes then?

File: guides/source/upgrading_ruby_on_rails.md [hunk 2]
```diff
@@ -535,7 +535,7 @@ ActiveSupport::JSON::Encoding.time_precision = 0
 Previously, Rails allowed inline callback blocks to use `return` this way:
 
 ```ruby
-class ReadOnlyModel < ActiveRecord::Base
+class ReadOnlyModel < ApplicationRecord
```
Comment (akshay-vishnoi) [inline guides/source/upgrading_ruby_on_rails.md]: @dhh - Same

File: guides/source/plugins.md (hunk 1 above)
Comment (dhh) [inline guides/source/plugins.md]: `ApplicationRecord < ApplicationRecord` doesn't make any sense?

Comment (akshay-vishnoi) [inline guides/source/plugins.md]: Oops. My Bad. Exclude it too.

File: guides/source/upgrading_ruby_on_rails.md (hunk 2 above)
Comment (akshay-vishnoi) [inline guides/source/upgrading_ruby_on_rails.md]: @dhh - Please reopen this PR. I have updated it with the changes only in this file.

**Comment by akshay-vishnoi** [comment]
Its not showing the new changes. Will I need to create a new PR or opening this PR will do the work?

**Pull request closed without merging**